cves/kernel/CVE-2017-1000252.yml

```diff
@@ -332,8 +332,8 @@ vouch:
 
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+  answer: true
+  note: engineers from both Amazon and Red Hat signed off on the fix.
```

Comment:
I got this comment on mine so I think other people are doing it also, list the names of the developers

Reply:
Thanks, I will update it to include it.